lighthouse-extension/gulpfile.js

```diff
@@ -94,61 +84,82 @@ gulp.task('chromeManifest', () => {
   .pipe(gulp.dest('dist'));
 });
 
-gulp.task('browserify', () => {
+function applyBrowserifyTransforms(bundle) {
+  // Fix an issue with Babelified code that doesn't brfs well.
```

Comment:
I don't remember what this comment means, but maybe remove since we don't use babel anymore?